integration-tests/src/test/java/oracle/weblogic/kubernetes/ItKubernetesNameSpaceWatchingEvents.java

```diff
@@ -0,0 +1,373 @@
+
+    logger.info("verify NamespaceWatchingStarted event is logged in {0}", domainNamespace5);
+    checkEvent(opNamespace, domainNamespace5, null, NAMESPACE_WATCHING_STARTED, "Normal", timestamp);
+
+    // verify there is no event logged in domainNamespace4
+    logger.info("verify NamespaceWatchingStarted event is not logged in {0}", domainNamespace4);
+    assertFalse(domainEventExists(opNamespace, domainNamespace4, null, NAMESPACE_WATCHING_STARTED,
+        "Normal", timestamp), "domain event " + NAMESPACE_WATCHING_STARTED + " is logged in "
+        + domainNamespace4 + ", expected no such event will be logged");
+
+    timestamp = now();
+    logger.info("Setting the domainNamesoaceRegExp to a new value {0}", domainNamespace4.substring(3));
+
+    // Helm upgrade parameters
+    opTestParams = opTestParams
+        .domainNamespaceSelectionStrategy("RegExp")
+        .domainNamespaceRegExp(domainNamespace4.substring(3));
+
+    upgradeAndVerifyOperator(opNamespace, opTestParams);
+
+    logger.info("verify NamespaceWatchingStopped event is logged in namespace {0}", domainNamespace5);
+    checkNamespaceWatchingStoppedEvent(opNamespace, domainNamespace5, null, "Normal", timestamp,
+        enableClusterRoleBinding);
+
+    logger.info("verify NamespaceWatchingStarted event is logged in namespace {0}", domainNamespace4);
+    checkEvent(opNamespace, domainNamespace4, null, NAMESPACE_WATCHING_STARTED, "Normal", timestamp);
+  }
+
+  /**
+   * Operator helm parameter domainNamespaceSelectionStrategy is set to Dedicated.
+   * If set to Dedicated, then operator will manage WebLogic Domains only in the same namespace which the operator
+   * itself is deployed, which is the namespace of the Helm release.
+   * Operator logs a NamespaceWatchingStopped in the operator domain namespace and
+   * NamespaceWatchingStopped event in the other domain namespaces when it stops watching a domain namespace.
+   *
+   * Test verifies NamespaceWatchingStopped event is logged when operator stops watching a domain namespace.
+   */
+  @Test
+  void testK8SEventsStartStopWatchingNSWithDedicated() {
+    OffsetDateTime timestamp = now();
+
+    // Helm upgrade parameters
+    OperatorParams opTestParams = opParamsOriginal;
+    opTestParams = opTestParams.domainNamespaceSelectionStrategy("Dedicated")
+                .enableClusterRoleBinding(false);
+
+    upgradeAndVerifyOperator(opNamespace, opTestParams);
+
+    logger.info("verify NamespaceWatchingStarted event is logged in {0}", opNamespace);
+    checkEvent(opNamespace, opNamespace, null, NAMESPACE_WATCHING_STARTED, "Normal", timestamp);
+
+    logger.info("verify NamespaceWatchingStopped event is logged in {0}", domainNamespace4);
+    checkNamespaceWatchingStoppedEvent(opNamespace, domainNamespace4, null, "Normal", timestamp, false);
+  }
+
+  /**
+   * Cleanup the persistent volume and persistent volume claim used by the test.
+   */
+  @AfterAll
+  public static void tearDown() {
```

Comment:
this method is not needed since no pv/pvc is created in this class

Reply:
removed